ext/src/http/client/curl/http_operation_curl.cc

```diff
@@ -453,7 +453,7 @@ void HttpOperation::Cleanup()
 #  define HAVE_TLS_VERSION
 #endif
 
-static long parse_min_ssl_version(std::string version)
+static int64_t parse_min_ssl_version(const std::string &version)
```

Comment:
Revert the long -> int64_t.
Keep the const reference.

Reply:
Noted.